test/e2e/cypress/e2e/hana_cluster_details.cy.js

```diff
@@ -342,6 +342,44 @@ context('HANA cluster details', () => {
       cy.wrap(user).as('user');
     });
 
+    describe('Check Execution', () => {
+      it('should forbid check execution when the correct user abilities are missing', () => {
+        cy.get('@user').then((user) => {
+          cy.createUserWithAbilities(user, []);
+          cy.login(user.username, password);
+        });
+
+        cy.visit(`/clusters/${availableHanaCluster.id}/settings`);
+
+        cy.contains('button', 'Start Execution').should('be.disabled');
+
+        cy.contains('button', 'Start Execution').click({ force: true });
+
+        cy.contains('span', 'You are not authorized for this action').should(
+          'be.visible'
+        );
+      });
+
+      it('should enable check execution button when the correct user abilities are present', () => {
+        cy.get('@user').then((user) => {
+          cy.createUserWithAbilities(user, [
+            { name: 'all', resource: 'cluster_checks_execution' },
+          ]);
+          cy.login(user.username, password);
+        });
+
+        cy.visit(`/clusters/${availableHanaCluster.id}/settings`);
```

Comment:
Here we could have `cy.contains('button', 'Start Execution').should('be.enabled');`